esp/bindings/http/platform/httpprot.cpp

```diff
@@ -426,16 +426,16 @@ bool CHttpThread::onRequest()
     Owned<ISecureSocket> secure_sock;
     if(m_is_ssl && m_ssctx)
     {
-        DBGLOG("Creating secure socket");
+        ESPLOG(LogMax, "Creating secure socket");
         secure_sock.setown(m_ssctx->createSecureSocket(m_socket.getLink(), getEspLogLevel()));
         int res = 0;
         try
         {
-            DBGLOG("Accepting from secure socket");
+            ESPLOG(LogMax, "Accepting from secure socket");
             res = secure_sock->secure_accept();
             if(res < 0)
             {
-                DBGLOG("Error accepting from secure socket");
+                ESPLOG(LogMax, "Error accepting from secure socket");
```

Comment:
This should have a lower log level... if an error occurs, it is imperative we report it.

Reply:
@rpastrana I agree, thanks for noticing! Changed it to LogMin.